src/OrbitGl/GlCanvas.cpp

```diff
@@ -31,8 +31,8 @@ float GlCanvas::kZValueTrack = 0.01f;
 float GlCanvas::kZValueIncompleteDataOverlayPicking = 0.02f;
 float GlCanvas::kZValueEventBar = 0.03f;
 float GlCanvas::kZValueBox = 0.05f;
-float GlCanvas::kZValueBoxBorder = 0.06f;
-float GlCanvas::kZValueEvent = 0.07f;
+float GlCanvas::kZValueEvent = 0.06f;
+float GlCanvas::kZValueBoxBorder = 0.07f;
```

Comment:
I'd like Pablo's opinion on this change. Could this affect other parts of the UI?

Or are these constants only specific to the thread dependencies feature?

Reply:
To me, it doesn't seem like these constants affected the UI when I ran the program. But I'm not sure if there are some small or very specific elements that have also changed. I did discuss this change with Pablo offline however and he okay-ed it.

Reply:
Great. In that case he should definitely be the one who should review this. :-)

Reply:
I proposed initially BoxBorder in the original PR (it makes sense for me) and I completely missed https://github.com/google/orbit/pull/3941.

This switch was the correct solution, BoxBorder should be on top of Event. I guess that when we introduced it in the histogram we didn't consider Events at all.